packages/checkout/sdk/src/types/config.ts

```diff
@@ -125,6 +125,13 @@ export type OnRampConfig = {
 export type ConnectConfig = {
   /** A boolean value for enabling/disabling WalletConnect */
   walletConnect: boolean;
+  /** A configuration for injected wallets */
+  injected: {
+    /** List for sorting injected wallets via wallet rdns */
+    priorityWalletRdns: string[];
+    /** List for blocking injected wallets via wallet rdns */
+    blacklistWalletRdns: string[];
+  }
```

Comment:
We will need to update the docs for the ConnectWidget with these config values, and maybe where to find the rdns values.